src/components/anti-invite-links.ts

```diff
@@ -50,7 +50,7 @@ export default class AntiInviteLinks extends BotComponent {
     }
 
     async handle_message(message: Discord.Message) {
-        if (this.wheatley.is_authorized_mod(message.author)) {
+        if (await this.wheatley.check_permissions(message.author, Discord.PermissionFlagsBits.Administrator)) {
```

Comment:
Mods don't have `Discord.PermissionFlagsBits.Administrator`, I think we want them to be immune to anti-invite-link

Reply:
Sure, or maybe even ModerateMembers